sdk/identity/azure-identity/azure/identity/_credentials/default.py

```diff
@@ -102,6 +104,7 @@ def __init__(self, **kwargs):
         managed_identity_client_id = kwargs.pop(
             "managed_identity_client_id", os.environ.get(EnvironmentVariables.AZURE_CLIENT_ID)
         )
+        interactive_browser_client_id = kwargs.pop("interactive_browser_client_id")
```

Comment:
`pop` with no default raises when the key doesn't exist
```suggestion
        interactive_browser_client_id = kwargs.pop("interactive_browser_client_id", None)
```